### ISSUE TITLE
Training parameters for Imagenet resnet-50

### ISSUE THREAD

**Issue by heejincs** [vlfeat/matconvnet#1009]
Hi, all.

 Thank you for the great library! Where can I find the training parameters for pretrained Imagenet resnet-50? 
 There are lots of training parameters for cnn_imagenet.m for instance learning rate that can influence to training.
 Could you provide the parameters used for the training? 
 For instance, in https://github.com/bazilas/matconvnet-ResNet provide the following parameters.
cnn_cifar('modelType', 'res', 'depth', 164, 'GPU', 1, 'batchSize', 128,'momentum', 0.95, 'weightDecay', 0.0005, 'Nclass', 100, 'learningRate', [0.01*ones(1,3) 0.1*ones(1,70) 0.01*ones(1,20) 0.001*ones(1,30)])
 Thank you.

Sincerely,
 Heejin Choi
